test/jsonValidationTest.js

```diff
@@ -201,7 +201,7 @@ function validateNoFromTemplateId(settings, file) {
 
 function validateCategory(category, file) {
```

Comment:
current test only checks existing category if it's correct, we'll need another to verify that it exist as well

Reply:
I believe that already exists: See validateJsonExistForWorkbook